FrontEnd/src/UserAuthentication/LoginPage.jsx

```diff
@@ -0,0 +1,44 @@
+import  { useNavigate } from "react-router-dom"
+import { useState } from "react";
+import {loginUser, logOutUser, auth, onAuthStateChanged} from "./Auth";
+
+function LoginPage(){
+    let navigate = useNavigate();
+    const [loginEmail, setLoginEmail] = useState('');
+    const [loginPassword, setLoginPassword] = useState('');
+    const [user, setUser] = useState({});
+    onAuthStateChanged(auth, (currentUser)=>{
+        setUser(currentUser);
+    })
+
+    const handleLogin = async(event) =>{
+        event.preventDefault();
+            try{
+                const result = await loginUser(loginEmail, loginPassword);
+                if (result !== null){
+                    navigate('/MainPage')
+                } else {
+                    console.error('Error creating user');
```

Comment:
Could you please explain what you mean by "throw a new exception"?

Reply:
@Oluwatosin12-bit This is how you throw exceptions in JS: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/throw

Since it's in try-catch, then you can catch all exceptions in the catch with one error log.